push

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+shopt nullglob
+
+OS=${OS:-fedora}
+VERSION=${VERSION:-40}
+OSVER=$OS-$VERSION
+WORKDIR=$PWD/work
+REPOSITORY=quay.io/foreman/nboci-files
+
+# create image index and set the annotation at the index level
+# TODO set the artifactType at the index level https://github.com/containers/podman/issues/23598
+podman manifest create "$REPOSITORY:$OSVER" --annotation org.pulpproject.netboot.version=1
```

Comment:
I don't know how much this script can be made reusable because it really depends on the destination repo and tagging pattern.
For example, fedora repo would presumably have:
quay.io/fedora/fedora-netboot:40
quay.io/fedora/fedora-netboot:40-amd64
quay.io/fedora/fedora-netboot:40-arm64

Centos would follow a different pattern like:
quay.io/centos/centos-netboot:stream9
quay.io/centos/centos-netboot:stream9-amd64

Reply:
Wouldn't it then be `OS=centos VERSION=stream9`? It's a good question and perhaps best to write a `Containerfile` to see if it would indeed be possible.

Reply:
Now added `Containerfile-centos-amd64` and `Containerfile-centos-arm64`. I at least verified this passes:
```
podman build -f Containerfile-centos-amd64 --build-arg version=stream10-development
podman build -f Containerfile-centos-amd64 --build-arg version=stream9
```

Reply:
So the plan for RHEL is to have y-stream releases here: 9.2, 9.3, 9.4 that is actually the goal of this.

I wonder how we should treat Stream 9 tho. I guess releasing on a monthly basis and tagging it with some YYYYMMDD format so we can then cleanup and garbage-collect all "older than 6 months" commits once we have some sort of a cleanup script would be a good approach, what ya think?